daily-diary/src/firebase.tsx

```diff
@@ -0,0 +1,35 @@
+import { initializeApp } from "firebase/app";
+import { addDoc, collection, getDocs, getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyBBFpbCPxTjwDx-AFolWCD73sYOKXfSqT0",
```

Comment:
don't commit these kinds of secrets, use the .env file to store these and make sure to add that file to .gitignore